### PR TITLE
Add part 6 of tutorial series

### PR DESCRIPTION
**Pull request by ghost**
See #155

Only https://github.com/WoltLab/docs.woltlab.com/pull/179/commits/07fbff519e31f3ac14c7e7b5ee318cbf7ff5a2ae is relevant for reviewing.

### CODE REVIEW

File: docs/tutorial/series/part_6.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+# Part 6: Activity Points and Activity Events
+
+In this part of our tutorial series, we use the person information added in the [previous part](part_5.md) to award activity points to users adding new pieces of information and to also create activity events for these pieces of information.
+
+
+## Package Functionality
+
+In addition to the existing functions from [part 5](part_4.md), the package will provide the following functionality after this part of the tutorial:
```
Comment (TimWolla) [inline docs/tutorial/series/part_6.md]: Link is inconsistent with the text.

File: docs/tutorial/series/part_6.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+- Users are awarded activity points awarded for adding new pieces of information for people.
```
Comment (TimWolla) [inline docs/tutorial/series/part_6.md]: Duplicate “awarded”.

File: docs/tutorial/series/part_6.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+which updates the number of instances for which any user received person information activity points.
+(This data worker also requires the phrases `wcf.acp.rebuildData.com.woltlab.wcf.people.information` and `wcf.acp.rebuildData.com.woltlab.wcf.people.information.description`).
+
+Second, we add an event listener for `UserActivityPointItemsRebuildDataWorker` to update the total user activity points awarded for person information:
+
+{jinja{ codebox(
+    language="xml",
+    title="eventListener.xml",
+    contents="""
+<eventlistener name=\"execute@wcf\system\worker\\UserActivityPointItemsRebuildDataWorker\">
```
Comment (TimWolla) [inline docs/tutorial/series/part_6.md:100]: Just wondering: Are the backslashes in front of the `"` actually required here? Does anything about the escape requirement change if we use a **r**aw string? The latter would also allow you to remove the duplicate backslash in front of the `u`.

```
r"""
foo
"""
```

Comment (ghost) [reply docs/tutorial/series/part_6.md:100]: I was unable to get raw strings working here.

**Review by TimWolla — APPROVED** [review]

**Pull request merged**